src/functions.jl

```diff
@@ -1,47 +1,38 @@
 using DataFrames
 """
-    npt_running_average(x_val, y_val, n = 5, x_name = "x", y_name = "y")
+    npt_running_average(x, y, n = 5, x_name = "x", y_name = "y")
```

Comment:
You could use type annotations in the function signature too, but I don't think it's necessary here, because it is clear from context that `y` has to be numeric.

... or is it? Perhaps say in the docstring that `x` can be a vector of any type, while `y` must be a numeric vector.